src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -339,7 +329,7 @@ function makeGeneticTrackDatum_Data_Type3(oncoprinterInputLine:OncoprinterInputL
     return ret as OncoprinterGeneticTrackDatum_Data;
 }*/
 
-export function makeGeneticTrackDatum_Data_Type2(
+export function makeGeneticTrackDatum_Data(
```

Comment:
was there a specific reason to have mixed case function names?

Reply:
I guess its just supposed to make these huge type names a little easier to comprehend